assets/src/settings-page/index.js

```diff
@@ -161,13 +161,46 @@ function Root( { appRoot } ) {
 	const onSubmit = useCallback( ( event ) => {
 		event.preventDefault();
 
+		/*
+		 * Adds the developer tools 'Validated URLs' & 'Error Index' on turning ON
+		 * the 'Enable Developer Tools' toggle button and saving.
+		 */
+		const addAmpDevOptions = ( optionTitle, optionUrl ) => {
+			const wpAmpSettings = document.querySelector( '#toplevel_page_amp-options ul' );
+
+			if ( wpAmpSettings === null ) {
+				return;
+			}
+
+			const listElement = document.createElement( 'li' );
+			const anchorElement = document.createElement( 'a' );
+			anchorElement.innerText = optionTitle;
+			anchorElement.setAttribute( 'href', optionUrl );
+			listElement.appendChild( anchorElement );
+			wpAmpSettings.appendChild( listElement );
+		};
```

Comment:
This would make sense to be defined at the top level of the module, rather than as a nested callback.